### PR TITLE
Remove profile association to sso tokens

### PR DESCRIPTION
**Pull request by Rotorsoft**
## Link to Issue
Closes: #8377

## Description of Changes
- Removes association
- Fixes schema

## Test Plan
-  Unit tests

### CODE REVIEW

File: libs/model/src/models/sso_token.ts [hunk 1]
```diff
@@ -23,19 +12,18 @@ export default (
...
     {
       id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
-      issued_at: { type: Sequelize.INTEGER, allowNull: true },
-      issuer: { type: Sequelize.STRING, allowNull: true },
-      address_id: { type: Sequelize.INTEGER, allowNull: true },
-      profile_id: { type: Sequelize.INTEGER, allowNull: true },
-      state_id: { type: Sequelize.STRING, allowNull: true },
+      issued_at: { type: Sequelize.INTEGER, allowNull: false },
+      issuer: { type: Sequelize.STRING, allowNull: false },
+      address_id: { type: Sequelize.INTEGER },
```
Comment (Rotorsoft) [inline libs/model/src/models/sso_token.ts:17]: @jnaviask are SSO tokens only applicable to magic addresses? ...seems like it

Should we make address_id a required field? (allowNulls = false) ... looks like we shouldn't have any records where address_id, issued_at, issuer are null ... can't find any in the dev db anyway

File: libs/schemas/src/entities/user.schemas.ts [hunk 2]
```diff
@@ -59,6 +59,16 @@ export const Address = z.object({
   updated_at: z.any(),
 });
 
+export const SsoToken = z.object({
+  id: PG_INT.optional(),
```
Comment (Rotorsoft) [inline libs/schemas/src/entities/user.schemas.ts:63]: @jnaviask can we have multiple SSO tokens for any given address id? This surrogate id can be removed and we can use the address_id as PK if that's the case.

File: packages/commonwealth/server/routes/authCallback.ts [hunk 3]
```diff
@@ -24,6 +24,8 @@ const authCallback = async (
   req: TypedRequestQuery<AuthCallbackReq>,
   res: TypedResponse<AuthCallbackResp>,
 ) => {
+  console.log('***********************authCallback', req.user);
```
Comment (Rotorsoft) [inline packages/commonwealth/server/routes/authCallback.ts:27]: @jnaviask @kurtisassad I'll remove this log before merging... just trying to trace here...logging in and out with magic or Metamask is not reaching this point.

Comment (Rotorsoft) [reply packages/commonwealth/server/routes/authCallback.ts:27]: @jnaviask after debugging with @kurtisassad, seems like this callback (including endpoint) has been deprecated and replaced by the Magic passport strategy. Can you confirm this before I try to remove it? 

If removed, the magic strategy above (`/server/passport/magic.ts`) is the only point where we create SsoToken records, and seems to confirm that issuer and address id fields are not optional.

File: libs/model/src/models/sso_token.ts (hunk 1 above)
Comment (jnaviask) [reply libs/model/src/models/sso_token.ts:17]: @Rotorsoft That is correct -- SSO tokens are only used by magic addresses and should always have an address_id.

File: libs/schemas/src/entities/user.schemas.ts (hunk 2 above)
Comment (jnaviask) [reply libs/schemas/src/entities/user.schemas.ts:63]: @Rotorsoft A given address_id should always only have one or (in most cases) no SsoTokens.

File: packages/commonwealth/server/routes/authCallback.ts (hunk 3 above)
Comment (jnaviask) [reply packages/commonwealth/server/routes/authCallback.ts:27]: @Rotorsoft This route is used by discord bot for external linking, cc @timolegros -- can we remove this?

File: libs/model/src/models/sso_token.ts (hunk 1 above)
Comment (jnaviask) [reply libs/model/src/models/sso_token.ts:17]: @Rotorsoft actually they're used to link addresses on Discord, but TBD the status of that feature.

File: packages/commonwealth/server/routes/authCallback.ts (hunk 3 above)
Comment (kurtisassad) [reply packages/commonwealth/server/routes/authCallback.ts:27]: For more context it was added in this PR:
https://github.com/hicommonwealth/commonwealth/pull/2099

Along with a call from the front end, but this call was later removed. But maybe it is used now for the discord bot.

**Review by timolegros — APPROVED** [review]

**Review by jnaviask — APPROVED** [review]

**Pull request merged**